test/source/tests/unit-node.ts

```diff
@@ -904,6 +904,30 @@ ${testConstants.smimeCert}`),
       t.pass();
     });
 
+    test('[unit][MsgUtil.isPasswordMesageEnabled] test password protected message compliance', async t => {
+      const disallowTerms = ['[Classification: Data Control: Internal Data Control]', 'droid'];
+
+      const subjectsToTestObj: { [key: string]: boolean } = {
+        '[Classification: Data Control: Internal Data Control] Quarter results': false,
+        'Conference information [Classification: Data Control: Internal Data Control]': false,
+        'Classification: Data Control: Internal Data Control - Tomorrow meeting': true,
+        'Internal Data Control - Finance monitoring': true,
+        // term check should work only for exact matches - if we have droid in the list of strings,
+        // password-protected messages shouldn't be disabled for subjects with Android word
+        'Android phone update': true,
+        'droid phone': false,
+        // Check for case insensitive
+        'DROiD phone': false,
+      };
```

Comment:
well done with this test 👍
let's also add mentioned case when disallowed term is surrounded by brackets